src/main/java/com/codahale/metrics/newrelic/NewRelicReporter.java

```diff
@@ -83,6 +95,7 @@
   @Override
   public synchronized void start(long initialDelay, long period, TimeUnit unit) {
     allMetricTransformers().forEach(registry::addListener);
+    LOG.info("New Relic Reporter " + implementationVersion + " is starting");
```

Comment:
Super nit-picky, but I think I'd put the version in parenthesis, or at least call out somehow that the version is what is being printed out.

Reply:
yah, done